Makefile

```diff
@@ -174,3 +175,34 @@ serve-dev: development.ini build .db/.setup-db-dev
 .PHONY: serve
 serve: development.ini build
 	$(VENV_BIN)/pserve $<
+
+.PHONY: docker-build
+docker-build:
+	docker build .
+
+.PHONY: docker-serve
+docker-serve: docker-build
+	PGHOST=$(PG_HOST) PGUSER=$(PG_USER) PG_DB=$(PG_DB) PGPASSWORD=$(PG_PASSWORD) docker-compose up -d
+
+.PHONY: docker-down
+docker-down:
+	PGHOST=$(PG_HOST) PGUSER=$(PG_USER) PG_DB=$(PG_DB) PGPASSWORD=$(PG_PASSWORD) docker-compose down
```

Comment:
I added them because even though it works without, you always get warnings:
```
WARNING: The PGHOST variable is not set. Defaulting to a blank string.
WARNING: The PGUSER variable is not set. Defaulting to a blank string.
WARNING: The PGPASSWORD variable is not set. Defaulting to a blank string.
WARNING: The PG_DB variable is not set. Defaulting to a blank string.
```

Reply:
ah ok